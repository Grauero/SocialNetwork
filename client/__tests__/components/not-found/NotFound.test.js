import React from 'react';
import { mount } from 'enzyme';
import toJSON from 'enzyme-to-json';

import NotFound from '../../../src/components/not-found/NotFound';

const component = mount(<NotFound />);

it('matches snapshot', () => {
  expect(toJSON(component)).toMatchSnapshot();
});
