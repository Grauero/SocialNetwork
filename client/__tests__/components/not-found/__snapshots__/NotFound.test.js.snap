// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<NotFound>
  <div>
    <h1
      className="display-4"
    >
      Page Not Found
    </h1>
    <p>
      Sorry, this page does not exist
    </p>
  </div>
</NotFound>
`;
